components/homme/src/test_mod.F90

```diff
@@ -75,15 +78,27 @@ subroutine set_test_initial_conditions(elem, deriv, hybrid, hvcoord, tl, nets, n
     case('mtest2');
     case('mtest3');
     case('held_suarez0');
-    case('jw_baroclinic');
+    case('hgw');
```

Comment:
just being picky -- these tests names are very short.

Reply:
I will make these names longer and more descriptive.